pandas/tests/frame/test_to_csv.py

```diff
@@ -943,6 +943,22 @@ def test_to_csv_compression(self, compression):
             with tm.decompress_file(filename, compression) as fh:
                 assert_frame_equal(df, read_csv(fh, index_col=0))
 
+    def test_to_csv_compression_size(self, compression):
```

Comment:
Since these are all the same tests I think it makes more sense to put in `test_common` and parametrize for the different writers, rather than splitting out across the various modules

Reply:
done.